### ISSUE TITLE
Composer2: 20k lines of problem description

### ISSUE THREAD

**Issue by glaubinix** [composer/composer#9068]
Is there maybe a way to shorten the problem description a bit in cases like this. It gets very hard to find the actualy problem in all the text.

My `composer.json`:

```json
{
    "repositories": [
        {
            "type": "package",
            "package": {
                "name": "test/other",
                "version": "1.0",
                "require": {
                    "vlucas/phpdotenv": "^2.4"
                }
            }
        }
    ],
    "require": {
        "laravel/framework": "^6.0",
        "test/other": "^1.0"
    }
}

```

Output of `composer diagnose`:

```
Checking composer.json: WARNING
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.
Checking platform settings: OK
Checking git settings: OK
Checking http connectivity to packagist: OK
Checking https connectivity to packagist: OK
Checking github.com oauth access: OK
Checking disk free space: OK
Checking pubkeys:
Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: OK
Composer version: 2.0-dev+d8fa7464334404643237971bdf921120f784916e
PHP version: 7.2.31
PHP binary path: /usr/local/Cellar/php@7.2/7.2.31_1/bin/php
OpenSSL version: OpenSSL 1.1.1g  21 Apr 2020
```

When I run this command:

```
composer update
```

I get the following output:

```                       ✘ 1
/t/test ❯❯❯ composer update
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

And here goes a bit over 20k lines of problem description which doesn't
fit into a GitHub issue but can be seen in the gist below
```
https://gist.github.com/glaubinix/5d16b4cff0c92c888025fa0fb00824d7

**Comment by mr-feek** [comment]
I'm running into this too! It takes several minutes alone just to output the string of errors 😂 

@Seldaek if you could provide some guidance into expected functionality here, I may be able to take a stab at implementing something.

Would the desire be to provide a new consolidated output mode hidden behind a flag, or somehow reduce the current output to something more actionable?

**Comment by Seldaek** [comment]
Should be fixed in 2.0 actually as we removed the learned rules output.
